src/core/load-fragment.c

```diff
@@ -1154,6 +1163,11 @@ int config_parse_exec_output(
                 }
         }
 
+        if (obsolete)
+                log_syntax(unit, LOG_WARNING, filename, line, 0,
```

Comment:
I think this should be LOG_NOTICE at most. With the amount of matches that @mbiebl posted, LOG_WARNING seems excessive.

Reply:
good point. in tmpfiles when we patch /var/run/ to /run/ we also use LOG_NOTICE:

https://github.com/systemd/systemd/blob/master/src/tmpfiles/tmpfiles.c#L2482